internal/datastore/elastic/elastic.go

```
@@ -46,25 +71,104 @@ func (e *ElasticRepository) Write(ctx context.Context, event entity.Event) error
+		result.Failures = append(result.Failures, UpdateFailure{
+			Index:      f.Index,
+			DocumentID: f.ID,
+			Cause:      f.Cause.Reason,
+		})
+	}
+
+	return result, nil
+}
+
+// buildUpdateScript creates a Painless script and params map from the updates
+func buildUpdateScript(updates map[string]interface{}) (string, map[string]interface{}) {
```

golangci-lint (2.12.2)

[error] func buildUpdateScript is unused

(unused)